parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -305,6 +305,20 @@ else
         # -________________-
         systemctl restart --no-block apt-daily.service
         aptmarkWALinuxAgent unhold &
+    elif [[ $OS == $MARINER_OS_NAME ]]; then
+        {{- if EnableUnattendedUpgrade }}
+        # By default the dnf-automatic is service is notify only in Mariner.
+        # Enable the automatic install timer and the check-restart timer.
+        # Stop the notify only dnf timer since we've enabled the auto install one.
+        # systemctlDisableAndStop adds .service to the end which doesn't work on timers.
+        systemctl disable dnf-automatic-notifyonly.timer
+        systemctl stop dnf-automatic-notifyonly.timer
+        # At 6:00:00 UTC (1 hour random fuzz) download and install package updates.
+        # Disable timer persistence so dnf-automatic-install doesn't run immediately on first boot.
+        sed -i 's/Persistent=true/Persistent=false/' /usr/lib/systemd/system/dnf-automatic-install.timer
```

Comment:
We should check if this is still necessary. Since the timer is disabled for image creation it shouldn't need to immediately fire on first boot.